README.md

```diff
@@ -121,81 +95,71 @@ Also in the output will be the endpoints that will be used to interface with the
 The default ws url is "127.0.0.1:8546" and the default http url is "127.0.0.1:8545".
 These values will be used as the `ethereum.wsPath` and `ethereum.httpPath` in the config, respectively.
 
-#### Bitcoin
-For Bitcoin, ipfs-blockchain-watcher is able to operate entirely through the universally exposed JSON-RPC interfaces.
-This means any of the standard full nodes can be used (e.g. bitcoind, btcd) as the data source.
+### Indexer
+Finally, setup the indexer process itself.
 
-Point at a remote node or set one up locally using the instructions for [bitcoind](https://github.com/bitcoin/bitcoin) and [btcd](https://github.com/btcsuite/btcd).
+Start by downloading ipld-eth-indexer and moving into the repo:
 
-The default http url is "127.0.0.1:8332". We will use the http endpoint as both the `bitcoin.wsPath` and `bitcoin.httpPath`
-(bitcoind does not support websocket endpoints, the watcher currently uses a "subscription" wrapper around the http endpoints)
+`GO111MODULE=off go get -d github.com/vulcanize/ipld-eth-indexer`
 
-### Watcher
-Finally, setup the watcher process itself.
-
-Start by downloading ipfs-blockchain-watcher and moving into the repo:
-
-`GO111MODULE=off go get -d github.com/vulcanize/ipfs-blockchain-watcher`
-
-`cd $GOPATH/src/github.com/vulcanize/ipfs-blockchain-watcher`
+`cd $GOPATH/src/github.com/vulcanize/ipld-eth-indexer`
 
 Then, build the binary:
 
 `make build`
 
 ## Usage
-After building the binary, run as
+After building the binary, three commands are available
+
+* Sync: Streams raw chain data at the head, transforms it into IPLD objects, and indexes the resulting set of CIDs in Postgres with useful metadata.
+
+`./ipld-eth-indexer sync --config=<the name of your config file.toml>`
+
+* Backfill: Automatically searches for and detects gaps in the DB; syncs the data to fill these gaps.
+
+`./ipld-eth-indexer backfill --config=<the name of your config file.toml>`
+
+* Resync: Manually define block ranges within which to (re)fill data over HTTP; can be ran in parallel with non-overlapping regions to scale historical data processing
+
+`./ipld-eth-indexer resync --config=<the name of your config file.toml>`
 
-`./ipfs-blockchain-watcher watch --config=<the name of your config file.toml>`
 
 ### Configuration
 
-Below is the set of universal config parameters for the ipfs-blockchain-watcher command, in .toml form, with the respective environmental variables commented to the side.
-This set of parameters needs to be set no matter the chain type.
+Below is the set of parameters for the ipld-eth-indexer command, in .toml form, with the respective environmental variables commented to the side.
+The corresponding CLI flags can be found with the `./ipld-eth-indexer {command} --help` command.
```

Comment:
is it full list of available parameters?

Reply:
It is!